modules/installation/proc-install-rhdh-orchestrator-helm.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies
+// assembly-orchestrator-helm-modules.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="proc-install-orchestrator-helm_{context}"]
+= Install Orchestrator Helm
+
+You can install the Orchestrator flavor of {product} ({product-very-short}) on OpenShift using Helm charts, enabling serverless workflows and workflow orchestration capabilities.
+
+.Prerequisites
+
+* You are logged in as an administrator on the OpenShift cluster.
+* You have installed Helm and configured locally.
```

Comment:
So this is entirely using the Helm CLI on OCP. Similar to https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.6/html/installing_red_hat_developer_hub_on_openshift_container_platform/assembly-install-rhdh-ocp-helm#proc-install-rhdh-ocp-helm-gui_assembly-install-rhdh-ocp-helm and https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.6/html/installing_red_hat_developer_hub_on_openshift_container_platform/assembly-install-rhdh-ocp-helm#proc-install-rhdh-ocp-helm-cli_assembly-install-rhdh-ocp-helm, I wonder if we should not also document deploying the Orchestrator flavor using the OCP web UI. WDYT?